src/legacy/components/AmplitudeAnalytics/index.ts

```diff
@@ -7,9 +7,9 @@ import { Identify, identify, init, track } from '@amplitude/analytics-browser'
  * member of the organization on Amplitude to view details.
  */
 export function initializeAnalytics(isDevEnvironment = process.env.NODE_ENV === 'development') {
-  const API_KEY = process.env.REACT_APP_AMPLITUDE_KEY
-  if (isDevEnvironment || !API_KEY) return
+  if (isDevEnvironment) return
```

Comment:
I added this cause testing locally in "production mode" would break herre, and i would think is not a mandatory param, right?

Reply:
Yes, I just inherited it from the previous branch

Reply:
I'm bringing this back, since it breaks the e2e https://github.com/cowprotocol/cowswap/pull/2733